wondrous-dev/wondrous-frontend
# adding hotkey to the org page

Adding tab+t as a hotkey to create task in Org Page

Test: 
- go to org page 
- press tab and t together, modal should appear to create task
